convert2rhel/checks.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright(C) 2016 Red Hat, Inc.
```

Comment:
Nit: we need to update the copyright year. Not related to the scope of this MR, just for reference

Reply:
ref #208